### DISCUSSION TITLE
New project manager

### DISCUSSION THREAD

**Discussion by HFolkertH** [cincheo/daquota#139] [Q&A] [answered]
Should it be possible to start a application from the new projectmanager?
 Now i can only change description and version numer there

**Comment by renaudpawlak** [comment]
Major changes indeed !!
Now you can create and save your projects on the cloud :)
First, let me point out that you can continue working exactly as before. Juste start a blank project and use "Import/Export snapshot" to load and write dLite files on the disk.

That said, I guess that you are talking about the "Tools > Project manager" dialog ? 
If yes, you are right, but at this point this dialog is mainly to allow deleting projects and versions. In order to run a project, use the Open project dialog and to create new version, you can save the project and switch on the "create new version" checkbox.

I hope it helps...

**Comment by renaudpawlak** [comment, marked as answer]
To be clear: it will be possible in the future, but right now there are many other things to be implemented before that :)

**Comment by HFolkertH** [comment]
Hi Renaud,

thanks for explaining. I am just trying out all the nice new features you have added to the platform lately. Very impressive! What is the roadmap you have in mind.? 
I am surprised that there are so few people using dLite. Are you building dLite for you and me only :-)

Have a nice weekend

folkert

**Reply by renaudpawlak** [reply]
Ah ah ! Thanks. This is only the beginning. I did not plan to have too many users yet ;) As you may have noticed, nowadays, if you don't advertise a little, it is complicated. People are very busy being solicited by products having the kind of money to mass-advertise. 

Although dLite is ready-to-use (please, use it ;)), I am still in the phase where I am building apps with dLite (for actual users/customers). Once these apps will be built, hopefully dLite will be user-friendly and stable enough to be advertised ;)

I believe that the best way to build a good useful software product is to actually use it while developing it. That's my strategy. Plus, I don't like advertising ;)

Here are some of the major features on the roadmap at this point (open-ended list, not counting all the improvements that will be made while developing the apps):

- A better UX with a "simple/advanced" mode switch so that anyone can use it (basic features) without training
- Support for custom components and templates
- Support for IndexDB
- More support for eco-design and accessibility
- Domain modelling support, with data models and workflows
- Administration tools
- ...
